modules/network-observability-quickfilter.adoc

```diff
@@ -0,0 +1,23 @@
+= Filtering network traffic
+By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:
+
+
+.Quick filters
+The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can select or select the options from console. 
+
+.Advanced filters
+To set advanced filters:
+
+* Select the parameters. The default selected parameter is *Common Namespaces*.
+* Enter the value. To filter the data that matches the text value exactly, enter the value in double quotation marks. 
+
+The section *Common* in the parameter drop-down applies filter to both source and destination. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

Comment:
Instead of
> applies filter to both source and destination.

I would say:
> applies filter to either source or destination.

(For example, "common namespace = foo" will match flows where source namespace=foo or dest namespace = foo, but it hasn't to be both at the same time)